app/components/card/card.scss

```diff
@@ -29,7 +29,7 @@
         }
     }
     &-footer {
-        padding: 0 40px;
+        padding: 0 20px;
```

Comment:
This only effect mobile as the divs inside are centered and not effected here.  Added so icon does not break at <340